MAUI/Autocomplete/UI-Customization.md

```diff
@@ -859,3 +859,61 @@ autocomplete.CursorPosition = 4;
 {% endtabs %}
 
 N> Two way binding not supported in Android platform.
+
+## Border visibility
+
+The `ShowBorder` property of [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) is used to modify the visibility of the border and its default value is **true**. The following code example demonstrates how to change the border visibility,
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfAutocomplete x:Name="autoComplete"
+                        WidthRequest="200" 
+                        HeightRequest="35"
+                        ShowBorder = "False" />
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Inputs;
+
+SfAutocomplete autocomplete = new SfAutocomplete();
+autocomplete.WidthRequest = 200;
+autocomplete.HeightRequest = 35;
+autocomplete.ShowBorder = false;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![ShowBorder](Images/UICustomization/ShowBorder.png)
+
+### TextAlignment
+
+The [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) provides support to customize the text alignment by using the `HorizontalTextAlignment` and `VerticalTextAlignment` properties.
+
+N> `HorizontalTextAlignment` dynamic changes is not working in Android platforms.
```

Comment:
Dynamic changes to the HorizontalTextAlignment property may not be functioning as expected on Android platforms.

Reply:
Addressed the comments.